src/ui/LayoutEditor.tsx

```diff
@@ -138,57 +138,110 @@ export class LayoutEditor extends React.Component<Props, State> {
+                                        The main component for visualizing all the split times. Each segment is shown in a tabular fashion showing the segment icon, segment name, the delta compared to the chosen comparison, and the split time. The list provides scrolling functionality, so not every segment needs to be shown all the time.
+                                    </span>
                                 </MenuItem>
-                                <MenuItem onClick={(_) => this.addComponent(LiveSplit.DetailedTimerComponent)}>
+                                <MenuItem divider />
+                                <MenuItem className="tooltip" onClick={(_) => this.addComponent(LiveSplit.DetailedTimerComponent)}>
                                     Detailed Timer
+                                    <span className="tooltip-text">
+                                        Shows two timers, one for the total time of the current run and one showing the time of just the current segment. Other information, like segment times of up to two comparisons, the segment icon, and the segment's name, can also be shown.
+                                    </span>
                                 </MenuItem>
-                                <MenuItem onClick={(_) => this.addComponent(LiveSplit.GraphComponent)}>
-                                    Graph
+                                <MenuItem className="tooltip" onClick={(_) => this.addComponent(LiveSplit.TimerComponent)}>
+                                    Timer
+                                    <span className="tooltip-text">
+                                        Shows the total time of the current run as a digital clock. The color of the time shown is based on a how well the current run is doing compared to the chosen comparison.
+                                    </span>
+                                </MenuItem>
+                                <MenuItem divider />
+                                <MenuItem className="tooltip" onClick={(_) => this.addComponent(LiveSplit.CurrentComparisonComponent)}>
+                                    Current Comparison
+                                    <span className="tooltip-text">
+                                        Shows the name of the comparison that the timer is currently comparing against.
+                                    </span>
                                 </MenuItem>
-                                <MenuItem onClick={(_) => this.addComponent(LiveSplit.PbChanceComponent)}>
+                                <MenuItem className="tooltip" onClick={(_) => this.addComponent(LiveSplit.CurrentPaceComponent)}>
+                                    Current Pace
+                                    <span className="tooltip-text">
+                                        Shows a prediction of the current run's final time. The remainder of the run is predicted based on the chosen comparison for the component.
```

Comment:
I thought it might be worth mentioning Best Possible Time since it's a really common use case.
```suggestion
                                        Shows a prediction for the current run's final time. The remainder of the run is predicted based on the chosen comparison for the component. For example, the "Best Segments" comparison can be chosen to show the best possible final time for the current run based on the Sum of Best Segments.
```